scripts/docker/dockerfile_gcc-8

```diff
@@ -1,25 +1,38 @@
 FROM axom/compilers:gcc-8
+LABEL maintainer="white238@llnl.gov"
+ARG branch=develop
 
+SHELL ["/bin/bash", "-c"]
 RUN sudo apt-get update -y
-RUN sudo apt-get install curl -fy
-
+RUN sudo apt-get install curl wget ssh pkg-config build-essential libopenblas-dev tar python3 -fy
 RUN sudo apt-get -qq install -y --no-install-recommends gfortran-7 && sudo update-alternatives --install /usr/bin/gfortran gfortran /usr/bin/gfortran-7 100
+RUN sudo ln -f -s python3 /usr/bin/python # Compatibility with legacy Python
 
-RUN git clone --recursive https://github.com/LLNL/axom.git
+# Avoid building CMake from scratch
+RUN wget https://cmake.org/files/v3.10/cmake-3.10.1-Linux-x86_64.sh
+RUN sudo sh cmake-3.10.1-Linux-x86_64.sh --prefix=/usr --exclude-subdir
 
-RUN git clone --recursive https://github.com/LLNL/RAJA.git
+# Avoid building all MPICH3.3 dependencies from scratch - is there a xenial-compatible binary somewhere?
+RUN wget http://www.mpich.org/static/downloads/3.3.2/mpich-3.3.2.tar.gz && tar xfz mpich-3.3.2.tar.gz
+RUN cd mpich-3.3.2 && ./configure -prefix=/usr && make -j && sudo make install
 
-WORKDIR "/home/axom/RAJA"
-RUN mkdir build
+# RUN sudo useradd -m -s /bin/bash -G sudo axom
 
-WORKDIR "/home/axom/RAJA/build"
-RUN cmake ../. -DENABLE_TESTS=OFF
-RUN make 
-RUN sudo make install
+WORKDIR "/home/axom"
+USER axom
 
-WORKDIR "/home/axom/axom"
-RUN ./scripts/uberenv/uberenv.py --spack-config-dir=./scripts/uberenv/spack_configs/docker --spec=%gcc@8.1.0 --prefix=/home/axom/axom_tpls -k
+RUN git clone --recursive --branch $branch --single-branch --depth 1 https://github.com/LLNL/axom.git
```

Comment:
I'm planning to look into this, but I'm assuming `$branch` is a user-settable variable within our new github action ?

Reply:
I found it -- 
![image](https://user-images.githubusercontent.com/5626552/100820142-6ee4cf00-3402-11eb-8714-72ebf3b2043c.png)